src/probes/isRequire.js

```diff
@@ -28,14 +25,35 @@ function validateNode(node, { tracer }) {
   ];
 }
 
+function validateNodeEvalRequire(node, options) {
+  const id = getCallExpressionIdentifier(node);
+  const argument = getCallExpressionArguments(node);
+  if (id === null) {
+    return [false];
+  }
+
+  return [
+    id === "eval" && argument[0] === "require",
+    options?.identifiersName[0]?.name ?? void 0
+  ];
+}
+
+
 function main(node, options) {
-  const { analysis } = options;
+  const { analysis, data: calleeName } = options;
   const { tracer } = analysis;
 
   if (node.arguments.length === 0) {
     return;
   }
   const arg = node.arguments.at(0);
+  const id = getCallExpressionIdentifier(node);
+
+
+  if (arg.value === "require" && id === "eval") {
```

Comment:
No need to reuse function here if calleeName already provide the information

Reply:
agreed i have removed this unecessary function call